### ISSUE TITLE
Package yoga-layout-prebuilt throws exception when trying to run tasks from the command line.

### ISSUE THREAD

**Issue by simple-nicholas** [burgerbuds/swiff#35]
Package yoga-layout-prebuilt throws exception when trying to run tasks from the command line.

Tested on fresh installs of swiff on fresh installs of nodejs versions 8.16.1, 10.16.3, 10.17.0, and 12.13.1.

Stack trace from trying to run '$ swiff --composerPull'
```
/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

TypeError: Cannot read property 'yogaNode' of undefined
    at squashTextNodes (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/render-node-to-output.js:47:38)
    at squashTextNodes (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/render-node-to-output.js:59:43)
    at renderNodeToOutput (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/render-node-to-output.js:134:18)
    at renderNodeToOutput (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/render-node-to-output.js:154:7)
    at renderNodeToOutput (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/render-node-to-output.js:154:7)
    at renderNodeToOutput (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/render-node-to-output.js:154:7)
    at Instance.node [as renderer] (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/renderer.js:108:37)
    at Instance.onRender (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/instance.js:105:14)
    at resetAfterCommit (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/ink/build/reconciler.js:24:14)
    at commitRootImpl (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/react-reconciler/cjs/react-reconciler.development.js:13367:5)
    at unstable_runWithPriority (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/scheduler/cjs/scheduler.development.js:643:12)
    at runWithPriority (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/react-reconciler/cjs/react-reconciler.development.js:1796:10)
    at commitRoot (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/react-reconciler/cjs/react-reconciler.development.js:13211:3)
    at runRootCallback (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/react-reconciler/cjs/react-reconciler.development.js:12346:20)
    at /Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/react-reconciler/cjs/react-reconciler.development.js:1844:24
    at unstable_runWithPriority (/Users/nicholas/.nvm/versions/node/v10.17.0/lib/node_modules/swiff/node_modules/scheduler/cjs/scheduler.development.js:643:12)
```

**Comment by ben-rogerson** [comment]
Hey Nicholas!
This sounds like a strange issue, particularly since it's been working previously in older versions of Node. I've finished up for the year (and without a dev setup right now) so I'm unable to attempt a bug reproduction.
Were you able to reproduce the bug on another workstation?

**Comment by Kashkin** [comment]
Hey Ben,

Alex and I encountered this error on his machine, when running Node v12.12.0.

It only occurs when running `swiff --taskName` in the CLI; running `swiff` will boot the interface as expected. The interface also has a bug; the left and right arrow keys don't work for moving between pages.

I'm still back on 10.16.2, so I haven't run into either issue myself.

Hope that helps.

**Comment by bighoho123** [comment]
This is related to the ink bug in the recent version.

I've updated the depedency of ink to a more strict version to avoid this.

https://github.com/vadimdemedes/ink/issues/241

The new version is released as `v1.5.2`

**Comment by ben-rogerson** [comment]
Nice one Jinzhe!

>
